src/index.ts

```diff
@@ -75,6 +83,11 @@ require('dotenv').config()
       describe: 'Upload generated collection to Postman, after conversion',
       type: 'boolean'
     })
+    .option('syncPostman', {
+      alias: 'syncPostman',
```

Comment:
feels like this could just be moved up above postmanFastSync

Reply:
re-organised the options